.travis.yml

```diff
@@ -14,3 +14,5 @@ cache:
 branches:
   only:
     - master
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
trailing new line.

You should configure your editor to add these automatically